### DISCUSSION TITLE
osu!lazer map selection idea and mod idea

### DISCUSSION THREAD

**Discussion by Plosmo** [ppy/osu#21283] [Ideas] [answered]
Idea1: here's some ideas for lazer's map selection:

Collection folders like windows file explorer to have well organized collections (aka being able to put a collection inside an other one

filters like you already did with the star rating slider, it would be better to have a menu similar to the mods menu with a slider for every value
it would be way faster and user friendly compared to using magical spells like " unplayed= star>6 ar>9.5 od>9 length>120".

Idea2: Mod 

This one is a complicated one to implement compared to the two other ideas. it would be kinda cool to have a mod similar to geometry dash's practice mod " if you die it makes you respawn a bit earlier" but with some mod settings like selecting what makes you die (breaking SS, missing, or dying)
the complicated part is the respawn mechanism, i could think of something like:
if death occurs during a kiai respawn is set at kiai's first pattern
if kiai is longer than 30 seconds or if death occurs outside kiai make user respawn 2 patterns earlier (ignore patterns having less than 4 notes) (if respawn is more than 30 notes away respawn at first pattern)
(when mapping combo resets i count that as a pattern)
that's the simplest code for the respawn that wouldnt be too broken that i could think of where it would respawn you at a non annoying part
respawn would be something like: it sets you back one pattern (minimum of 4 notes) earlier than your actual respawn point so you can get back to the rythm without that pattern actually counting in the score.

Those ideas were both from 
Yt: Tonio DND
osu!: Toniowo

**Comment by ghost** [comment, marked as answer]
Please search before suggesting new ideas.

Duplicate of #21281
